tests/Integration/inc/classes/CDN/RocketCDN/APIClient/getSubscriptionData.php

```diff
@@ -4,11 +4,11 @@
 
 use WPMedia\PHPUnit\Integration\TestCase;
 use WPMedia\PHPUnit\Integration\ApiTrait;
-use WP_Rocket\CDN\RocketCDN\APIClient;
+use WP_Rocket\Engine\CDN\RocketCDN\APIClient;
```

Comment:
Move this test and all of the RocketCDN integration tests into `tests/Integration/inc/CDN/RocketCDN/...`.